### DISCUSSION TITLE
Symfony 6.0.0  "An error occurred while loading the web debug toolbar"

### DISCUSSION THREAD

**Discussion by mlcrm** [symfony/symfony#44412] [Q&A] [answered]
My controller method looks like this
```
    public function edit(User $user, Request $request): Response
    {
        dump($user);
        return $this->render('base.html.twig');
    }
```
When trying to dump an object in the controller, an error occurs web-profiler ["An error occurred while loading the web debug toolbar"]
In the log file the following error ["Symfony\Component\HttpKernel\Debug\FileLinkFormatter::$requestStack must not be accessed before initialization"]
I assume this is a bug in the new version. Is it so?

**Comment by norkunas** [comment, marked as answer]
See https://github.com/symfony/symfony/issues/44401 and https://github.com/symfony/symfony/pull/44402
